### PR TITLE
Fixes for CentOS guests

### PR DESCRIPTION
**Pull request by olavph**
- Fix os variant in libvirt tests
- Allow executing tests with nested virtualization
- Show installation logs
- Increase memory for unattended_install.url test, similar to RHEL

### CODE REVIEW

**Comment by olavph** [comment]
Any feedback on this?

File: virttest/test_setup.py [hunk 1]
```diff
@@ -2052,15 +2052,20 @@ def disable_smt(params=None):
...
                                            % cmd_output[1])
     else:
-        smt_output = process.system_output(cmd, shell=True).strip()
-    if (smt_output != "SMT is off"):
+        try:
+            smt_output = process.system_output(cmd, shell=True).strip()
+        except process.CmdError as info:
+            smt_output = info.result.stderr.strip()
+            if smt_output not in SMT_DISABLED_STRS:
+                raise
```
Comment (sathnaga) [inline virttest/test_setup.py]: better to callout proper error, `exceptions.TestSetupFail...`

Comment (olavph) [reply virttest/test_setup.py]: Agreed and done.

**Comment by olavph** [comment]
@sathnaga I've added some snipppets of the install test logs to commit message of https://github.com/avocado-framework/avocado-vt/pull/1215/commits/727a4aca3b39ac959e1d5757080341d520acbfe3 to clarify what was missing, is that what you had in mind?
```
Below are some snippets of the actual installation logs that are now shown.
They help in identifying which step is currently being executed by Anaconda and
debugging when the installlation fails (for example, it shows yum failures).

2017-12-14 16:56:17: Welcome to ESC[0;31mCentOS Linux 7 (AltArch)ESC[0m!
...
2017-12-14 16:56:27: ESC[1;23rStarting installer, one moment...ESC[1;24rESC[H
...
2017-12-14 16:56:34: ESC[1;23rESC[HESC[2Banaconda 21.48.22.121-1 for CentOS AltArch 7 started.ESC[1;24rESC[HESC[3;54H
...
2017-12-14 18:27:44: OS install is completed
```

**Comment by sathnaga** [comment]
@olavph I meant by just install pass output with this patchset...

**Comment by olavph** [comment]
The install test is passing just like before:
```
 (1/2) io-github-autotest-qemu.unattended_install.url.http_ks.default_install.aio_native: PASS (5909.25 s)
```
The long execution time is due to nested virtualization.

**Pull request merged**